FacebookStrategy.php

```diff
@@ -63,7 +63,12 @@ public function int_callback(){
 			parse_str($response, $results);
 
 			if (!empty($results) && !empty($results['access_token'])){
-				$me = $this->me($results['access_token']);
+				
+				$fields = array();
```

Comment:
@Kryniol , the idea is configure 'fields' as settings, so you can set comma separated fields like this :

<?php
'Facebook' => array(
    'app_id' => 'YOUR APP ID',
    'app_secret' => 'YOUR APP SECRET'
    'scope' => 'email',
    'fields' => 'id,first_name,last_name,gender,email', 
)
I had write a little aditional info at README
https://github.com/jsenin/facebook/blob/master/README.md

It's the minor change that i need to keep it working.

greets